frontend/src/component/common/PermissionIconButton/PermissionIconButton.tsx

```diff
@@ -13,7 +13,7 @@ import {
 } from 'hooks/useHasAccess';
 
 interface IPermissionIconButtonProps {
-    permission: string;
+    permission: string | string[];
```

Comment:
Allows `PermissionIconButton` to evaluate multiple permissions, just like `PermissionButton`.